api/models/fixtures/multi_balancer.json

```diff
@@ -91,10 +91,28 @@
         "Ref": "WebPort80Balancer"
       }
     },
+    "WebPort80BalancerName": {
+      "Value": {
+        "Ref": "AWS::StackName"
```

Comment:
I've looked into this and the balancer name is not something that can be fetched directly. http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb.html

It looks like extracting it from the DNSName will require regular expressions. You can't just split on hyphens because it could be multiple words or the app name could have a hyphen in it. I'm not sure if it's possible to use a regex in this way, and it's not clear to me from the docs. Do you know?